src/main/resources/com/mathworks/ci/MatlabBuilder/runMatlabTests.m

```diff
@@ -68,6 +85,84 @@
     end
 end
 
+% Produce Cobertura model coverage report (Not supported below R2018a) 
+if produceModelCoverage
+    if ~exist('sltest.plugins.ModelCoveragePlugin', 'class')
+        warning('MATLAB:testArtifact:cannotGenerateModelCoverageReport', ...
+                'Unable to generate Cobertura model coverage report. To generate the report, use a Simulink Coverage license with MATLAB R2018a or a newer release.');
+    else 
+        import sltest.plugins.ModelCoveragePlugin;
+        import matlab.unittest.plugins.codecoverage.CoberturaFormat;
+        
+        mkdirIfNeeded(resultsDir);
+        coverageFile = fullfile(resultsDir, 'coberturamodelcoverage.xml');
+        runner.addPlugin(ModelCoveragePlugin('Producing',CoberturaFormat(coverageFile)));
+    end
+end
+
+% Produce PDF test report (Not supported below R2016b)
+if producePDFReport && ~exportSTMResults
+    if ~testReportPluginPresent
+        issuePDFReportUnsupportedWarning();
+    else
+        import matlab.unittest.plugins.TestReportPlugin;
+        mkdirIfNeeded(resultsDir);
+        
+        if stmResultsPluginPresent
+            runner.addPlugin(TestManagerResultsPlugin);
+        end
+        runner.addPlugin(TestReportPlugin.producingPDF(getPDFFilePath(resultsDir)));
```

Comment:
If the order doesn't matter, it may be more readable to add the pdf plugin near the import statement.